app/views/eventRegistrations.html

```diff
@@ -463,7 +479,7 @@ <h2 class="page-title" translate>Registrations</h2>
         </thead>
         <tbody>
           <tr
-            ng-repeat-start="r in registrants | orderBy:answerSort:reversesort"
+            ng-repeat-start="r in registrants | orderBy:answerSort"
```

Comment:
I also removed ```reverseSort``` as it was doing an additional layer of reverse sorting on the frontend, even though the API is handling that now. It was essentially undoing the reverse sorting from the API.